tests/fixtures/00-config-sample.toml

```diff
@@ -1,9 +1,8 @@
 [identity]
 # Node group: used for cluster-wide reboot orchestration (via remote strategy)
 group = "workers"
-
-# Node UUID: by default derived from "/etc/machine-id"
```

Comment:
Was dropping this comment intentional?

Reply:
Yes, this is only a fixture file used for testing. I took a note to add a sample with comments under `dist/`, will add on next rebase.